rotkehlchen/tasks/manager.py

```diff
@@ -891,6 +893,24 @@ def _maybe_delete_past_calendar_events(self) -> Optional[list[gevent.Greenlet]]:
             database=self.database,
         )]
 
+    def _maybe_query_graph_delegated_tokens(self) -> Optional[list[gevent.Greenlet]]:
+        """
+        Search for DelegationTransferredToL2
```

Comment:
This tells me nothing. This is not a graph specific file. You need a bit more detailed docstring